inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/StringMatchingRecommender.java

```diff
@@ -182,25 +182,23 @@ public void predict(RecommenderContext aContext, CAS aCas) throws Recommendation
             Collection<AnnotationFS> tokens = selectCovered(tokenType, sentence);
             for (AnnotationFS token : tokens) {
                 Trie<DictEntry>.Node node = aDict.getNode(text, token.getBegin());
-                if (node != null) {
+                if (node != null &&  tokens.stream().filter(t -> t.getEnd() == token.getBegin()+node.level).findAny().isPresent()) {
```

Comment:
Code not properly formatted (spaces missing). Please use checkstyle and eclipse code formatting style as indicated in the [developer instructions](https://inception-project.github.io//releases/0.12.2/docs/developer-guide.html#_checkstyle_and_formatting).

Reply:
Yes